### PR TITLE
Refactor singepoint2

### PR DESCRIPTION
**Pull request by yurii-klymenko**
review, approve, do not merge yet.

### CODE REVIEW

File: stix_shifter_modules/qradar/entry_point.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from .stix_transmission.qradar_query_connector import QRadarQueryConnector
+from .stix_transmission.qradar_status_connector import QRadarStatusConnector
+from .stix_transmission.qradar_delete_connector import QRadarDeleteConnector
+from .stix_transmission.qradar_results_connector import QRadarResultsConnector
+from .stix_transmission.arielapiclient import APIClient
+
+class EntryPoint(EntryPointBase):
+
+    def __init__(self, connection={}, configuration={}, options={}):
+        super(EntryPoint, self).__init__(options)
```
Comment (mdazam1942) [inline stix_shifter_modules/qradar/entry_point.py]: i think we can declare like this in python 3-

`super().__init__(options)`

File: stix_shifter/stix_translation/stix_translation.py [hunk 2]
```diff
@@ -73,8 +64,10 @@ def translate(self, module, translate_type, data_source, data, options={}, recur
         
         try:
             try:
-                translator_module = importlib.import_module(
-                    "stix_shifter_modules." + module + ".stix_translation." + module + "_translator")
+                connector_module = importlib.import_module("stix_shifter_modules." + module + ".entry_point")
+                entry_point = connector_module.EntryPoint(options=options)
```
Comment (mdazam1942) [inline stix_shifter/stix_translation/stix_translation.py]: we could still call this `interface`. just my opinion.

Comment (yurii-klymenko) [reply stix_shifter/stix_translation/stix_translation.py]: no interface anymore, all interface files are delete in next commits

File: stix_shifter/scripts/stix_shifter.py [hunk 3]
```diff
@@ -150,29 +151,29 @@ def main():
...
-            options = json.loads(options)
-
-        module, dialects = process_dialects(args_module_dialects, options)
-        args.options = json.dumps(options)
-
-        all_modules = module_list()
-        if not module in all_modules:
-            print(f"module '{module}' is not found")
-            help_and_exit = True
+  if 'module' in args:
```
Comment (mdazam1942) [inline stix_shifter/scripts/stix_shifter.py]: @yurii-klymenko I get an indentation error in this line. if you see the same error-
`Traceback (most recent call last):
  File "main.py", line 1, in <module>
    from stix_shifter.scripts.stix_shifter import main
  File "/Users/mdazam/uds_workspace/connector/stix-shifter/stix_shifter/scripts/stix_shifter.py", line 154
    if 'module' in args:
                       ^
IndentationError: unindent does not match any outer indentation level`

you may need to properly tab this line.

Comment (yurii-klymenko) [reply stix_shifter/scripts/stix_shifter.py]: it is fixed by Danny in v3-master

**Pull request merged**